### DISCUSSION TITLE
Recommended way to update poetry itself?

### DISCUSSION THREAD

**Discussion by codeananda** [python-poetry/poetry#9126] [Q&A]
Just realised that new versions of poetry have come out since I started using it for my project. 

I can't find any mention in the docs about how to update poetry itself. 

Do you just re-install it using any method [here](https://python-poetry.org/docs/#installation)?

**Comment by abn** [comment]
You can view update methods in the release announcement for your convenience.

https://python-poetry.org/blog/announcing-poetry-1.8.0/

**Comment by codeananda** [comment]
`poetry self update`

That's what I was looking for! Would be good to include this in the docs e.g. [here](https://github.com/python-poetry/poetry/blob/main/docs/_index.md)
